presto-product-tests/bin/run.sh

```diff
@@ -7,6 +7,12 @@ function absolutepath() {
     [[ $1 = /* ]] && echo "$1" || echo "$PWD/${1#./}"
 }
 
+# If profile is singlenode-ldap, import the root certificate of presto-coordinator
+if [ "$PROFILE" = "singlenode-ldap" ]
```

Comment:
In case of singlenode-ldap profile, we want the application runner ( container running cli ) to have the root certificate. Instead of creating a new container with that certificate, I am importing the certificate at run time for ldap-profile. I want to avoid creating a new docker image.  Also I can't import the certificate on the base image, as the base image is common for presto-master and application-runner (centos6-java8-oracle)